test/include/test_helper/test_helper.h

```diff
@@ -40,9 +40,14 @@ class TestHelper {
     static constexpr char E2E_TEST_FILES_DIRECTORY[] = "test/test_files";
     static constexpr char SCHEMA_FILE_NAME[] = "schema.cypher";
     static constexpr char COPY_FILE_NAME[] = "copy.cypher";
+    static constexpr char PARQUET_TEMP_DATASET_PATH[] = "dataset/parquet_temp/";
 
     static std::string getTmpTestDir() { return appendKuzuRootPath("test/unittest_temp/"); }
 
+    static std::string appendParquetDatasetTempDir(const std::string& dataset) {
+        return TestHelper::appendKuzuRootPath(TestHelper::PARQUET_TEMP_DATASET_PATH + dataset);
```

Comment:
Better to call `FileUtils::joinPath` to concatenation file paths?
Also in `appendKuzuRootPath`.